storybook/tailwind.config.js

```diff
@@ -136,7 +137,7 @@ module.exports = {
 		  '37xl':'37.5rem',
 		  '8xl':'8.8125rem',
 		  '4sxl':'4.1875rem',
-		  '18xl':'18.5rem',
+		  '50xl':'50rem',
```

Comment:
what is this value being used for and adjusted to config?

Reply:
i need them to adjust container instead of using inline style

Reply:
changes made to config has to be essential to the design if you will use an inline style for it once or twice only it is fine.
if you will use it several times then it could be added to the config, please let me know how many times it was used and on which components

Reply:
yes this one of the options but i did the other option thanks